src/viewers/PictureInPictureViewer.js

```diff
@@ -63,12 +63,11 @@ const PictureInPictureViewer = props => {
     transparentColor,
     onViewStateChange
   } = props;
-  const viewState =
-    initialViewState ||
+  const baseViewState =
+    viewStatesProp.find(v => v.id === DETAIL_VIEW_ID) ||
```

Comment:
Needs checking because `.find` is called but `viewStatesProp` could be undefined.

Reply:
Good catch!